demo/b2share_demo/helpers.py

```diff
@@ -455,14 +466,15 @@ def _process_record(rec):
         community = Community.get(name='CLARIN')
         result['community']=str(community.id)
     else:
-        return None
+        raise Exception("Community not found for domain: `{}`".format(rec['domain']))
```

Comment:
Are we sure we want to raise an exception here? Not just log the issue and continue?